src/styles/utils/_variables.scss

```
@@ -1,4 +1,4 @@
-$daysPerRow: 7;
+$daysPerRow: 8;
```

what-why?
7 days was perfectly correct
```suggestion
$daysPerRow: 7;
```

---

Ooops forgot to revert. Was testing. Thanx